store/migration/10003__ghost_integration.sql

```diff
@@ -0,0 +1,9 @@
+-- task_dag describes task dependency relationship
+-- from_task_id is blocked by to_task_id
+CREATE TABLE task_dag (
```

Comment:
let's include id, created_ts, updated_ts and payload column

payload might be needed if we want to annotate the edge

Reply:
Done.